modules/solid_mechanics/src/bcs/Pressure.C

```diff
@@ -89,7 +89,10 @@ PressureTempl<is_ad>::initialSetup()
     block_ids.insert(bids.begin(), bids.end());
   }
 
-  _coord_type = _fe_problem.getCoordSystem(*block_ids.begin());
+  if (block_ids.size())
+    _coord_type = _fe_problem.getCoordSystem(*block_ids.begin());
```

Comment:
it seems like in debug mode we should check whether the coord system is the same for all specified blocks

Reply:
This class certainly does not work if there are coordinate systems that are not matching. As is done in Navier-Stokes there are potentially additional pressure terms in RZ. They could change the class to not store a single `_coord_type` but until then there needs to be an error check. What I failed to realize is that if I expand the file the error check is already right below